llvm/lib/Target/SPIRV/SPIRVGlobalRegistry.cpp

```diff
@@ -1056,6 +1056,11 @@ SPIRVGlobalRegistry::getSPIRVTypeForVReg(Register VReg,
   return nullptr;
 }
 
+SPIRVType *SPIRVGlobalRegistry::getResultType(Register VReg) {
```

Comment:
Let's please change this a bit to take into account translation steps when ASSIGN_TYPE internal opcode is still present. To make this call more general it'd be enough to use https://github.com/llvm/llvm-project/blob/ebb3508899c3e1773884cf5bc1b1df6f32450ca9/llvm/lib/Target/SPIRV/SPIRVUtils.cpp#L639-L644 instead of calling `CurMF->getRegInfo().getVRegDef(VReg)`.

Reply:
Thanks. I did not know about that set of functions. I'll look there first from now on.